packages/validator/test/plugins/validation/2and3/paths.test.js

```diff
@@ -1,312 +0,0 @@
-const expect = require('expect');
```

Comment:
I actually started working on a new 'equivalent-paths' rule that would replace this legacy built-in rule, but found that the spectral:oas 'path-params' rule already performs this check.